src/MockWebServer.php

```diff
@@ -86,7 +86,15 @@ public function start() {
 			throw new Exceptions\ServerException("Error starting server, received '{$this->pid}', expected int PID");
 		}
 
-		sleep(1); // just to make sure it's fully started up, maybe not necessary
+		for( $i = 0; $i <= 20; $i++ ) {
+			usleep(100000);
```

Comment:
I sleep 100ms from the start because on my decently fast machine _every single execution_ for otherwise was a retry.

Reply:
Excellent approach! It's a good Kabbalistic number. lol